cfpb/sbl-frontend
# Render markdown

Closes #421 

## Changes

- Adds `react-markdown` library
- Successfully renders markdown from Submission's validation description
- Renders a more comprehensive markdown test at route `/markdown` (must be signed-in).

## How to test this PR

1. Upload a file for `123456789TESTBANK123` that has warnings (sbl-validations-pass.csv)
2. Visit http://localhost:8899/markdown
3. Observe the markdown from the Validation's description is rendered as a list item

## Notes
- We would need to add a plugin to add support for advanced elements like Tables or Strikethrough, but I've followed up with the Backend team to understand which Markdown elements we should expect from the API and only basic (list, bold) are expected to be used, so we're good to go with the current basic configuration. 

## Screenshots
<img width="1062" alt="react-markdown-test" src="https://github.com/cfpb/sbl-frontend/assets/2592907/6553343d-1978-434e-9cbd-b89a21b012cd">
